### PR TITLE
refine and fix typos for english version

### PR DESCRIPTION
**Pull request by doombeaker**

### CODE REVIEW

File: en/docs/quick_start/lenet_mnist.md [hunk 1]
```diff
@@ -278,10 +283,10 @@ def eval_job(
     return (labels, logits)
 ```
 
-Above is the coding of evolution job function and return object is a `Tuple[tp.Numpy, tp.Numpy]`. Tuple have two `numpy`  in it. We called the job function and computation accuracy according to return value.
+Code above shows defining a job function for evaluation and its return type is `Tuple[tp.Numpy, tp.Numpy]`. Tuple have two `numpy`  in it. We will call the job function and calculate the accuracy according to the return values.
 
-### Iteration evaluation
-The sample amount and correct sample amount in `acc`. We will called job function to get `labels` and `logits`：
+### Process of evaluation
+The `acc` function is used to count the total number of samples and the number of correct prediction results. We will call the job function to get paramters `labels` and `logits`:
```
Comment (MARD1NO) [inline en/docs/quick_start/lenet_mnist.md:289]: 我觉得更简洁点，就是acc函数是用来计算准确率的
The `acc` function is used to calculate the accuracy in training process

File: en/docs/quick_start/quickstart_in_3_min.md [hunk 2]
```diff
@@ -74,44 +69,47 @@ if __name__ == "__main__":
...
 
+* `load_data(BATCH_SIZE)`: Data loading;
 
+* `loss = train_job(images, labels)`: Return the loss value of each iteration;
 
+* `if i % 20 == 0: print(loss)`: Print a loss value once every 20 iteration;
 
-This page is a demonstration of a simple network. In Using [convolution neural network for handwriting recognition](lenet_mnist.md), we will give a more comprehensive and detailed introduction of using OneFlow process. In addition, you can reference to the training of all kinds of problems in detail in [Based topics](../basics_topics/data_input.md) of OneFlow.
+This page is a just a simple example on neural network. 
+A more comprehensive  and detailed introduction of OneFlow can be found in [Convolution Neural Network for Handwriting Recognition](lenet_mnist.md).
```
Comment (MARD1NO) [inline en/docs/quick_start/quickstart_in_3_min.md:108]: 多了一个空格

File: en/docs/basics_topics/build_nn_with_op_and_layer.md [hunk 3]
```diff
@@ -48,28 +54,29 @@ def lenet(data, train=False):
...
 
-The network constructing process and running process is separated in fact. The build process is OneFlow underlying calculation chart based on the description of the network connection relation and the process, but the real calculation happened at run time.
+## Data block in neural network
+As mentioned above, it is inaccurate to say that "data is used as the input" in that figure. Actually, when we define a network, there is no data in 'data', it's just a placeholder.
 
-When we build the network, we only describe the attributes and shapes(such as `shape`, `dtype`) of the nodes in network.There is no value in node, we call the node as **PlaceHolder**, OneFlow can compile and infer based on these placeholders to get the final computed graph. 
+ The construction process and running process of network in OneFlow are actually separate. The construction process is a process in which OneFlow builds the calculation graph according to the description of network defined by job function, but the real calculation happens at run time.
```
[the remark this answers is not in the document]
Comment (doombeaker) [reply en/docs/basics_topics/build_nn_with_op_and_layer.md:67]: seperate可以的

File: en/docs/basics_topics/data_input.md [hunk 4]
```diff
@@ -147,22 +148,22 @@ if __name__ == "__main__":
     images, labels = test_job()
     print(images.shape, labels.shape)
 ```
-In order to run the above code, we need a ofrecord dataset. We can [ load and prepare OFRecord dataset](../extended_topics/how_to_make_ofdataset.md) or we have a package which have 64 images called  [part-00000](https://oneflow-public.oss-cn-beijing.aliyuncs.com/online_document/docs/basics_topics/part-00000) .
+A ofrecord dataset is required to run the code above. We can making it by refering to [load and prepare OFRecord dataset](../extended_topics/how_to_make_ofdataset.md) or download the small prepared dataset which containing 64 images [part-00000](https://oneflow-public.oss-cn-beijing.aliyuncs.com/online_document/docs/basics_topics/part-00000) .
```
[the remark this answers is not in the document]
Comment (doombeaker) [reply en/docs/basics_topics/data_input.md:151]: 嗯，应该是an

File: en/docs/quick_start/lenet_mnist.md (hunk 1 above)
Comment (doombeaker) [reply en/docs/quick_start/lenet_mnist.md:289]: 这个后面统一减少吧，两个版本稳定了，我来统一瘦身下，并统一风格

File: en/docs/basics_topics/optimizer_in_function_config.md [hunk 5]
```diff
@@ -66,7 +66,7 @@ Use `config.default_logical_view(flow.scope.consistent_view())` to set the defau
 
 The `oneflow.function_config()` usually sets options related to computing resources, devices, and cluster scheduling. In contrast, we should set the optimizer, learning rate and hyperparameters inner the `job function`. 
 
-## Sample configuration
+## Exaple of configuration
```
Comment (MARD1NO) [inline en/docs/basics_topics/optimizer_in_function_config.md:69]: Exaple -> Example
